conscious/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "cvn",
+  "assets": [
+    {
+      "description": "Cvn is a Layer-1 blockchain built to deliver on the promise of DeFi",
+      "denom_units": [
+        {
+          "denom": "acvn",
+          "exponent": 0
+        },
+        {
+          "denom": "cvn",
+          "exponent": 18
+        }
+      ],
+      "base": "acvn",
+      "name": "Cvn",
+      "display": "cvn",
+      "symbol": "CVN",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cvn/images/cvn.svg",
```

Comment:
Do we have higher quality images to use here? They appear quite pixelated and blurry in an unintentional way.